packages/psss/src/components/EditableTable.js

```diff
@@ -52,16 +62,22 @@ export const EditableCell = React.memo(({ id, columnKey, ...props }) => {
 
   if (editable) {
     return (
-      <EditableTextField
-        id={id}
-        name={id}
-        defaultValue={defaultValue}
-        inputProps={{ 'aria-label': columnKey }}
-        error={!!errors[id]}
-        inputRef={register({
-          required: 'Required',
-        })}
-      />
+      <RedTooltip title={errors[id] ? errors[id].message : ''} placement="left" open>
+        <EditableTextField
+          id={id}
+          name={id}
+          defaultValue={defaultValue}
+          inputProps={{ 'aria-label': columnKey }}
+          error={!!errors[id]}
+          inputRef={register({
+            required: 'Required',
+            pattern: {
+              value: /^\d+$/,
```

Comment:
Any decimal points in PSSS?

Reply:
A good question but definitely not at this point, as we are counting numbers of people, so I think this  is a pretty solid assumption for now.